src/test/java/gameframework/game/GameDataTest.java

```diff
@@ -0,0 +1,41 @@
+package gameframework.game;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class GameDataTest {
+	
+	
+	GameConfiguration aGameConfiguration;
+	GameData aGameData;
+	Game aGame;
+	
+	@Before
+	public void init(){
+		aGameConfiguration = new GameConfiguration();
+		aGameData = new GameData(aGameConfiguration);
+		aGame = new GameDefaultImpl(aGameData);
+	}
+	
+	@Test
+	public void decreaseLife(){
+		int livesDeleted = 1 ;
+		int livesExpected = aGameConfiguration.getDefaultNbLives() - livesDeleted;
+		assertEquals((int)aGameData.getLife().getValue(), aGameConfiguration.getDefaultNbLives());
+		aGameData.decreaseLife(livesDeleted);
+		assertEquals(livesExpected, (int)aGameData.getLife().getValue());
```

Comment:
`livesExpected` -> `expectedLives`. The adjective always comes first in English.